src/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -157,7 +157,7 @@ protected function render( $attributes, $content, $block ) {
 							if ( $processor->next_tag( 'img' ) ) {
 								$processor->set_attribute(
 									'data-wc-on--click',
-									'actions.woocommerce.thumbnails.handleClick'
+									'actions.woocommerce.thumbnails.dialog.openDialog'
```

Comment:
I believe that when changing the `handleClick` for the dialog, the editor also changed this that is related to the `thumbnails` by mistake and it causes the Thumbnails to not be clickable anymore.